src/widgets/unfold_widgets/preprocessing_widget.py

```diff
@@ -62,10 +63,9 @@ def __init__(self, parent: QWidget, engine):
         self.auto_reduction_group.setTitle("Reduce dimensions automatically")
```

Comment:
group widget is called `Reduce dimensions automatically` and the two options inside it are `Reduce` and `Reduce automatically`. it's confusing, we should probably change these names and/or add a description label/tooltip indicating what the buttons are for